operator_ui/__tests__/containers/SignIn.test.js

```diff
@@ -47,6 +47,8 @@ describe('containers/SignIn', () => {
     await syncFetch(wrapper)
     const newState = store.getState()
     expect(newState.authentication.allowed).toEqual(true)
+
+    await syncFetch(wrapper)
```

Comment:
This is the first time I've looked at the operator UI code. There seems to be some obscure state management going on here with `syncFetch`. Looks harmless, though.

Reply:
This is due to async rendering issues. This new one is caused by the new `async function`